js-my-first-app/README.md

```diff
@@ -0,0 +1,13 @@
+# My First App
+
+This project is intended to be used along with our [Your First Klotho App](https://klo.dev/docs/tutorials/your_first_klotho_app). The purpose of this project is to show how you can quickly create a cloud-ready REST API with data persistence.
```

Comment:
are we not adding any klotho instructions and solely relying on the tutorial?

Reply:
> Is this the same as the getting started app? Is this just to provide a template for them to use? Are there docs side things to update to ref this?

Yes, the goal is to have them clone this repo and compile the app instead of writing the app step-by-step. I'm working with Ala on revisions to the tutorial.